server/src/main/java/com/objectcomputing/checkins/services/kudos/KudosServicesImpl.java

```diff
@@ -121,10 +122,6 @@ public Kudos save(KudosCreateDTO kudosDTO) {
 
     @Override
     public Kudos approve(Kudos kudos) {
-        if (!currentUserServices.isAdmin()) {
-            throw new PermissionException(NOT_AUTHORIZED_MSG);
-        }
-
         UUID kudosId = kudos.getId();
```

Comment:
Do we not need to replace this with a permission check? I don't see it happening elsewhere in here, but I could be missing it.

Reply:
The controller handles this:

```
    @Put
    @RequiredPermission(Permission.CAN_ADMINISTER_KUDOS)
    public Kudos approve(@Body @Valid Kudos kudos) {
        return kudosServices.approve(kudos);
    }
```